tests/test_integrated_channels/test_canvas/test_client.py

```diff
@@ -46,7 +47,7 @@ def setUp(self):
             oauth_api_path=self.oauth_api_path,
         )
         self.enterprise_config = canvas_factories.CanvasEnterpriseCustomerConfigurationFactory(
-            key=self.client_id,
+            client_id=self.client_id,
```

Comment:
this test will become live and valid when I run the last piece (migration to remove the key field in the next pr)